lib/assessment.rb

```diff
@@ -20,20 +31,43 @@
 # Name and location should be write-only, but age should be read-only.
 
 ##
-# your code here
+class Person
+  attr_writer :name
+  attr_reader :age
+  attr_writer :location
+
+
+  def initialize(name, age, location)
+
+    @name = name
+    @age = age
+    @location = location
+  end
+end
 ##
 
 # Question 4
 # Create a new Person object using parameters "Dave", 32, and "Ohio".
 # Then, set that Person's location to "Somerville"
 
-Answer.dave = nil # replace nil with your code
-Answer.dave.location = nil # replace nil with your code
+Answer.dave = dave = Person.new("Dave", 32, "Ohio") # replace nil with your code
+Answer.dave.location = dave.location.new = dave.location.gsub(/Ohio/, 'Somerville') # replace nil with your code
 
 # Question 5
 # Create another class called Developer that inherits from Person.
 # Give it a new public method called 'hire_for_job'
 
 ##
-# your code here
+# Should I assume that class Person already exists?  If not
+
+class Person
+  attr_reader :something
+def something
+end
+
+class Developer < Person
+  def hire_for_job
+  end
 ##
```

Comment:
You're missing an `end`. It would have been nice to see hire_for_job do something, but that wasn't required.